Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
Ok, tested. 
-SmingFramework builds. 
-Basic_rBoot builds. Tested on physical device, it boots OTA works, all working. 

Any other Sming example failing. 
As example: 
19:00:03 **** Build of configuration Sming for project Basic_Serial ****
make all 
C+ app/application.cpp
In file included from c:/tools/Sming/Sming/SmingCore/SmingCore.h:46:0,
                 from app/application.cpp:2:
c:/tools/Sming/Sming/SmingCore/Network/rBootHttpUpdate.h:14:23: fatal error: rboot-api.h: No such file or directory
 #include <rboot-api.h>
                       ^
compilation terminated.
make: *** [out/build/app/application.o] Error 1

19:00:04 Build Finished (took 557ms)

I recall this was expected as side effect of the new Makefile.